pkg/registration/spoke/options.go

```diff
@@ -22,9 +23,10 @@ type SpokeAgentOptions struct {
 	// 2. BootstrapKubeconfigSecret is the secret, a event handler will watch it, if the secret is changed, then rebootstrap.
 	// 3. BootstrapKubeconfigs is a list of file path, the controller uses one of its item to build the client.
 	// BootstrapKubeconfigs can only be used when MultipleHubs is enabled.
-	BootstrapKubeconfig       string
-	BootstrapKubeconfigSecret string
-	BootstrapKubeconfigs      []string
+	BootstrapKubeconfig             string
+	BootstrapKubeconfigSecret       string
+	BootstrapKubeconfigs            []string
+	BootstrapKubeconfigEventHandler *bootstrapKubeconfigEventHandler
```

Comment:
The SpokeAgentConfig might be a more suitable location for the BootstrapKubeconfigEventHandler, as it seems that SpokeAgentOptions primarily contains options and configurations

Reply:
Moved to `SpokeAgentConfig`.